apps/src/jigsaw/levels.js

```diff
@@ -131,8 +132,8 @@ module.exports = {
     },
     backgroundHSV: [184, 1.0, 0.733],
     ghost: {
-      x: 400,
-      y: 100,
+      x: 380,
```

Comment:
Is it worth storing these as constants and referencing them so that if they ever need to be updated in the future, they don't need to be updated in so many places?